src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -540,26 +540,18 @@ class TablePumpDescription:
     .. include:: /alfacase_definitions/list_of_unit_for_volume_fraction.txt
     .. include:: /alfacase_definitions/list_of_unit_for_pressure.txt
     """
-    speeds: Array = attr.ib(
-        default=Array([0.0] * 12 + [400.0] * 12 + [600.0] * 12, 'rpm'), validator=instance_of(Array)
-    )
-    void_fractions: Array = attr.ib(
-        default=Array(([0.0] * 6 + [0.1] * 6) * 3, '-'), validator=instance_of(Array)
-    )
-    flow_rates: Array = attr.ib(
-        default=Array([0.0, 0.05, 0.1, 0.15, 0.2, 0.3] * 6, 'm3/s'), validator=instance_of(Array)
-    )
+    speeds = attrib_array(Array([0.0] * 12 + [400.0] * 12 + [600.0] * 12, 'rpm'))
+    void_fractions = attrib_array(Array(([0.0] * 6 + [0.1] * 6) * 3, '-'))
+    flow_rates = attrib_array(Array([0.0, 0.05, 0.1, 0.15, 0.2, 0.3] * 6, 'm3/s'))
 
-    pressure_boosts: Array = attr.ib(
-        default=Array(
+    pressure_boosts = attrib_array(Array(
```

Comment:
I would expect black to nag about this, but OK.

Reply:
You are right, how odd... 🤔

Reply:
is black used for the alfasim-sdk tho? It never asked me to use it regardless of how "unformatted" the code seemed

Reply:
It is:

https://github.com/ESSS/alfasim-sdk/blob/fb-ASIM-4459-equipment-list-annulus-description/.pre-commit-config.yaml#L17-L24

Locally you need to run `pre-commit install` *once*, and from that point on, every commit will run the pre-commit checks (which includes black).

But regardless we use pre-commit.ci, which runs a pre-commit job here which would automatically format your code. That's why that's curios.

Reply:
To be clear: don't worry about it, we were just wondering, but there's no action to be taken on your side.

Reply:
yup, I figured that, since the jobs passed. I just find it odd, since black complains all the time with my commits fo other reps, but never had a problem with the sdk.

Reply:
Ahh it might be that `pre-commit.ci` pushes commits fixing the checks when possible. 😉

Reply:
Not a request for a change now.

If we move the `fmt: off/on` directive to inside the big array it is restricted to that structure.

[black playgound example](https://black.vercel.app/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4AbuApNdAD2IimZxl1N_WjIxhE5qLYty7VbcV-doEYjUTVrPHaS-uCA8Z9ilUzlI7F2mWB9tE62n0WHvpjGZ-_pUQA-S00-d8Wba_E7lsT2L-rwi7OU5cAQ_-29fkZ6ssmSz5xUcyYTKb4IU83N4Uv7DFehDXHmf0bziDxNQh_q8WeK3H2vzsACpO06lXAvhsqwlwoUErseUW6POcIWMK2IYwaAkStCvDAbtN-JC-vKLPqe7jC2rleQ85yjKP5IrOuf-l4bWg_Xs-KO3vNbjAwktbTKB6tRr6D3Yk5rfqWegenXMKfZozHtsERFHLN9J2fymqU5c8ZJ0pcznd61HLBB5Zc4H_iSf5gT6xeebbm91kwJ2I23ZlUthiJ6a_oZ_9GXoZU10gKmJTbcWXnRWJ8nkw457Mj2er3W-y1xNkZTcT1Denf2ols2y09k6_TAOY0-HOCYMmBIRTw3n_iHRId1PchlKKTExr9nJ02mV3cBW5TgHs_r-jwKhlCyGmylTXDo8qDqngXPDKHMroca_hjtKvHvUdnRMoHnEb4YsEuQrlZbcJXi5y5ajxB_VMXx0_7RlS60Ps14y1oXZmeK5o-q1OUnFX9Uck-JODWbNgMSRUWJG5zZhAKc2IWJPW7nxf7UJzqE1KxlIYIa6jGXANmMqG5LwyBbLVF0mnbS4rhgS5QxE-xKzw94A0Uhj6CFb9ADe1IqAf9wDafX7KwlPIa2wmnASGO_yQVCtXxSVt7YGbDlJWqYJj_cDanpfc5E5zSCi65ZU2hf9qQNRQl7Co_2mzcVEjfx4f4Ztbx3iOcOQJ45JyoAsiynvWq_KgsJ-r6AXUmJtRW3HvKq2YdlEJMhkQ_DwzNhsHzZxXTnmtLDBrO3I9xe29KU0AACM1RlkhxGijQABrwXvDQAAqzo7ZbHEZ_sCAAAAAARZWg==)